lib/oids.js

```diff
@@ -69,6 +69,7 @@ _IN('1.2.840.113549.1.9.14', 'extensionRequest');
 _IN('1.2.840.113549.1.9.20', 'friendlyName');
 _IN('1.2.840.113549.1.9.21', 'localKeyId');
 _IN('1.2.840.113549.1.9.22.1', 'x509Certificate');
+_IN('1.2.840.113549.1.9.16.2.14', 'timeStampToken');
```

Comment:
This should be sorted up a few lines.  I assume this is useful for an unauthenticated attributes use case?
```suggestion
_IN('1.2.840.113549.1.9.16.2.14', 'timeStampToken');
_IN('1.2.840.113549.1.9.20', 'friendlyName');
_IN('1.2.840.113549.1.9.21', 'localKeyId');
_IN('1.2.840.113549.1.9.22.1', 'x509Certificate');
```

Reply:
Fixed